utils/_context/core.py

```diff
@@ -189,6 +189,7 @@ def _init_scenario(self):
             self.weblog_env["DD_INSTRUMENTATION_TELEMETRY_ENABLED"] = "true"
         elif self.scenario == "APPSEC_IP_BLOCKING":
             self.proxy_state = '{"mock_remote_config_backend": "ASM_DATA"}'
+            self.weblog_env["DD_APPSEC_RULES"] = None
```

Comment:
it means that we won't test any chenge made on appsec rules file here (other way to says that this file is meaning less in this scenario). Is it waht you intend to do ?

Reply:
What this is aiming is that the env `DD_APPSEC_RULES` is not set when testing remote configuration `IP Blocking

Reply:
The point is that we tests the very last version of the rules file on it's `1main` branch (it's job called `dev` in the CI). To achieve that, we use `DD_APPSEC_RULES`. 

With your change, it means that the scenario `APPSEC_IP_BLOCKING` will use production rules file instead. 

It can make sense, but I just want to be sure you are aware of this detail. CC @nizox